internal/backend/httpsvc/intercept.go

```diff
@@ -51,7 +54,54 @@ func (r *responseInterceptor) Flush() {
 
 type RequestLogExtractor func(*http.Request) []zap.Field
 
-func intercept(z *zap.Logger, cfg *LoggerConfig, extractors []RequestLogExtractor, next http.Handler) (http.HandlerFunc, error) {
+var metrics = struct {
+	counters  map[string]metric.Int64Counter
+	durations map[string]metric.Int64Histogram
+}{make(map[string]metric.Int64Counter), make(map[string]metric.Int64Histogram)}
+
+func updateMetric(ctx context.Context, t *telemetry.Telemetry, path string, statusCode int, duration time.Duration) (err error) {
+	if !strings.HasPrefix(path, "/autokitteh.") {
+		return nil // only internal service APIs
+	}
+
+	slashParts := strings.Split(path, "/")
```

Comment:
could you add a comment with the "algorithm" here ?
not sure I get why you split and check 3 parts and then lower and search for . and do so on...

Reply:
added comments to the code

Reply:
why not just setting the path as a label?

Reply:
ok